Makefile

```diff
@@ -136,6 +136,13 @@ $(BINDIR)/lister-gen:
 	  --extra-peer-dirs k8s.io/kubernetes/pkg/api,k8s.io/kubernetes/pkg/api/v1,k8s.io/kubernetes/pkg/apis/meta/v1,k8s.io/kubernetes/pkg/conversion,k8s.io/kubernetes/pkg/runtime \
 	  -O zz_generated.conversion
 	# the previous three directories will be changed from kubernetes to apimachinery in the future
+	# Generate the internal clientset (pkg/client/clientset_generated/internalclientset)
+	$(DOCKER_CMD) $(BINDIR)/client-gen \
+		--input-base "github.com/kubernetes-incubator/service-catalog/pkg/apis/" \
+		--input servicecatalog/ \
```

Comment:
Just wondering... if the trailing slash isn't there you said things will go nuts - would it make sense to submit a PR that fixes the "nuts" aspect? Even if it just adds the trailing / for them when there is no "/" in the string at all.